rpc-jobs/multi-node-aio.yml

```diff
@@ -1,13 +1,26 @@
 - job:
-    name: 'OnMetal_Multi_Node_AIO_Prepare_Deployment'
+    name: 'OnMetal_Multi_Node_AIO'
     project-type: workflow
     build-discarder:
         days-to-keep: 30
     concurrent: true
     parameters:
       - string:
-          name: NODE
-          description: "Name or label of Jenkins node to run on"
+          name: REGION
```

Comment:
I was thinking about that too, but the OnMetal jobs will have different defaults for image and flavor and potentially region in the future if the non-OnMetal move from IAD to a different region.